### ISSUE TITLE
Display via DisplayPort issue

### ISSUE THREAD

**Issue by BerndVP** [theracermaster/Gigabyte-GA-Z77X-macOS-Install#53]
**System information**
 - macOS 10.12.4
 - GA-Z77X macOS Install v2.0.3 (2.0.5)
 - GA-Z77X-UD5H
 - Intel Core i7-3770K

**Issue description**
So, two days ago I was trying different resolutions of my Dell 27 inch monitor (2560x1440).
It's connected via DisplayPort.

Now when booting, you see the Sierra load bar the first one, then the second one, but when entering the Desktop, the screen stays black. Only when I connect a second screen to the DVI port, I get the first screen via DisplayPort working. Via HDMI I don't get it working and the screen resolution is blur.

If I disconnect the DVI when it's working, it stays working.
But every time I reboot the desktop I need a second screen connected via DVI or I can't get the first one working.

Is this a issue in Sierra or do I have to search somewhere else ?

Don't want to reinstall the system.

**Comment by theracermaster** [comment]
Was the display previously working over DisplayPort? Did it only stop working after you changed the resolution? You should be able to reset display preferences by deleting ~/Library/Preferences/ByHost/com.apple.windowserver.*.plist.

**Comment by BerndVP** [comment]
Yeah, DisplayPort worked perfect before I tried to change the resolution.

Just deleted the Plist file and rebooted the system without the DVI Display connected, but no luck.

**Comment by BerndVP** [comment]
Now if I disconnect the DVI Screen when in MacOS the 'main display' goes again to black

**Comment by BerndVP** [comment]
So, I attached a RTF file with the logs of 'console' where you can see that I get an error with the screen.

[Display Error Dell.rtf.zip](https://github.com/theracermaster/Gigabyte-GA-Z77X-macOS-Install/files/924476/Display.Error.Dell.rtf.zip)

**Comment by theracermaster** [comment]
Not sure if there's much I can do at this point, as this seems to be an issue that also affects some real Macs. Maybe SwitchResX or some similar software can reset the display preferences. FWIW, I haven't had any issues with DisplayPort output using Intel HD Graphics 4000, although I've only done 2560x1080.

**Comment by BerndVP** [comment]
Hi,
No problem.
I did a clean install of my system.
I used installer 2.0.5 but it always gave an error with the FakeSMC so I downloaded the latest FakeSMC on the repo of 'Rehabman', replaced it and that I got into the installer.
Now I see there is a V2.0.6 so no idea if this fix the 'FakeSMC' error.

**Comment by theracermaster** [comment]
What FakeSMC errors did you encounter?

**Comment by BerndVP** [comment]
![img_2191](https://cloud.githubusercontent.com/assets/8302682/25524187/d4dd39be-2c09-11e7-8ec4-c5602228bdf1.JPG)
